src-docs/src/views/code/code_example.js

```diff
@@ -39,6 +41,23 @@ export const CodeExample = {
   title: 'Code',
   intro: (
     <>
+      <EuiCallOut title="EUI provides support for static code">
+        <p>
+          <strong>EuiCode</strong> and <strong>EuiCodeBlock</strong> are
+          intended to render static lines or blocks of code in read-only
+          contexts.
+          <br />
+          If you need capabilities to edit, or want to print long code (e.g.,
```

Comment:
```suggestion
          contexts. If you need capabilities to edit, or want to print long code (e.g.,
```

Force line-breaks cause weird line-wraps.